src/App.vue

```diff
@@ -404,6 +401,9 @@ export default class App extends Mixins(AuthApiMixin, CommonMixin, DateMixin, Fi
     this.$root.$on('go-to-dashboard', () => {
       this.goToDashboard()
     })
+
+    // init app
+    this.onRouteChanged()
```

Comment:
Previously, due to "immediate: true", onRouteChanged() was run before created(). Now, it is run after created().